sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosDatabaseProperties.java

```diff
@@ -18,7 +18,7 @@
  * more cosmos items. Since databases are an an administrative resource and the Service Key will be required in
  * order to access and successfully complete any action using the User APIs.
  */
-public final class CosmosDatabaseProperties extends ResourceWrapper{
```

Comment:
`getId` is a required method in the Properties classes that we removed.
please add back here and other places.

Reply:
added back